app/editor/src/features/admin/report-templates/templates/body/CustomReport.cshtml

```diff
@@ -82,9 +82,10 @@
     {
       @* TABLE OF CONTENTS SECTION *@
       var tocCount = 0;
-      @foreach (var tableSection in Sections.Where(s => new [] {ReportSectionType.Content, ReportSectionType.Gallery, ReportSectionType.Text}.Contains(s.Value.SectionType)))
+      @foreach (var tableSection in Sections.Where(s => new [] {ReportSectionType.Content, ReportSectionType.Gallery, ReportSectionType.Text, ReportSectionType.Image}.Contains(s.Value.SectionType)))
```

Comment:
This is the change to the template.  It will now use the InTableOfContents to determine if it will show.  If the value is null it will default to show.